onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql

```diff
@@ -8,6 +13,10 @@ SELECT d.rowid,
        d.assaytype,
        d.container,
        d.totalTests
+<<<<<<< HEAD
+FROM  StudyDetails_RandalData s, "/ONPRC/Core Facilities/Genetics Core/MHC_Typing".geneticscore.mhc_data d
+=======
 
 FROM  StudyDetails_RandalData s, Site.{substitutePath moduleProperty('ONPRC_EHR','MHC_Container')}.geneticscore.mhc_data d
+>>>>>>> origin/release21.7-SNAPSHOT
```

Comment:
Does this line marked as a sql comment like adding "---" three  hyphen symbol preceding your text?  If not, then this will show as an error as sql statement.

Reply:
Update submitted that cleared the issues.  Also added update for MHC Data as it also contained the comment section.  WIll validate in Test and Provide results here

Reply:
Update submitted that cleared the issues.  Also added update for MHC Data as it also contained the comment section.  WIll validate in Test and Provide results here